src/abstractdataframe/abstractdataframe.jl

```diff
@@ -886,15 +892,40 @@ julia> filter(row -> row[:x] > 1, df)
 ├─────┼───────┼────────┤
 │ 1   │ 3     │ b      │
 │ 2   │ 2     │ a      │
+
+julia> filter(:x => >(1), df)
```

Comment:
As noted in another PR, I'd rather use something which doesn't involve the fancy syntax:
```suggestion
julia> filter(:x => x -> x > 1, df)
```

Also show an example with several columns?

Same below.